assets/js/atomic/blocks/product-elements/add-to-cart-form/style.scss

```diff
@@ -4,4 +4,9 @@
 		font-size: var(--wp--preset--font-size--large);
 	}
 	width: unset;
+
+	.input-text {
+		font-size: var(--wp--preset--font-size--small);
+		padding: 0.9rem 1.1rem;
```

Comment:
Last week, I was collaborating with @danielwrobert, and we discovered that the inputs were appearing small because the CSS class woocommerce-page was missing. We were able to resolve the issue by simply adding that class to the Add to Cart Form block wrapper. However, having a woocommerce-page class attached to a specific block seemed odd, so Dan suggested replacing the styles in the block's CSS instead.

Looking at the WooCommerce file that you shared, [this is the piece of code](https://github.com/woocommerce/woocommerce/blob/95ca53675f2817753d484583c96ca9ab9f725172/plugins/woocommerce/client/legacy/css/woocommerce-blocktheme.scss#L203-L206) that we are using. I'll be adding a comment to these lines of code.